wpilibc/src/main/native/include/frc/DriverStation.h

```diff
@@ -420,6 +420,23 @@ class DriverStation : public ErrorBase {
    */
   void WakeupWaitForData();
 
+  /**
+   * Allows the user to specify whether they want joystick connection warnings
+   * to be printed to the console. This setting is ignored when the FMS is
+   * connected -- warnings will always be on in that scenario.
+   *
+   * @param silence Whether warning messages should be silenced.
+   */
+  void SilenceJoystickConnectionWarnings(bool silence);
+
+  /**
+   * Returns whether joystick connection warnings are silenced. This will
+   * always return false when connected to the FMS.
+   *
+   * @return Whether joystick connection warnings are silenced.
+   */
+  bool AreJoystickConnectionWarningsSilenced() const;
```

Comment:
Do we use the "Are" nomenclature anywhere else?  This might be better as a singular--e.g. `IsJoystickConnectionWarningSilenced()`

Reply:
And change the setter to be singular as well if we do that.